AEPMessaging/Tests/UnitTests/MessagingRulesEngine+CachingTests.swift

```diff
@@ -162,37 +162,14 @@ class MessagingRulesEngineCachingTests: XCTestCase {
         XCTAssertFalse(mockCache.setCalled)
     }
 
-    /// The below tests for private func `cachePropositions` are executed via
-    /// internal methods `setPropositionsCache` and `clearPropositionsCache`
-    func testCachePropositionsClearCache() throws {
-        // test
-        messagingRulesEngine.clearPropositionsCache()
-
-        // verify
-        XCTAssertTrue(mockCache.removeCalled)
-        XCTAssertEqual("propositions", mockCache.removeParamKey)
-    }
-
-    func testCachePropositionsClearCacheThrows() throws {
-        // setup
-        mockCache.removeShouldThrow = true
-
-        // test
-        messagingRulesEngine.clearPropositionsCache()
-
-        // verify
-        XCTAssertTrue(mockCache.removeCalled)
-        XCTAssertEqual("propositions", mockCache.removeParamKey)
-    }
-
-    func testCachePropositionsSetCache() throws {
+    func testCachePropositionsAddCache() throws {
         // setup
         let propString: String = JSONFileLoader.getRulesStringFromFile("showOnceRule")
         let decoder = JSONDecoder()
         let propositions = try decoder.decode([PropositionPayload].self, from: propString.data(using: .utf8)!)
         
         // test
-        messagingRulesEngine.setPropositionsCache(propositions)
+        messagingRulesEngine.addPropositionsToCache(propositions)
```

Comment:
In the tests for `addPropositionsToCache`, we should also check that the `inMemoryPropositions` are updated.